### PR TITLE
Logo line break

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: convert.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 for f in source_files:
     subprocess.call(["inkscape", f, "--export-plain-svg=generated/" + os.path.splitext(f)[0] + ".svg"])
 
-subprocess.call(["webicon.sh", "sunpy_icon.svg"])
+subprocess.call(["./webicon.sh", "sunpy_icon.svg"])
```
Comment (dpshelio) [inline convert.py:28]: wouldn't be better instead to use: 
```python
subprocess.call(["bash", "webicon.sh", "sunpy_icon.svg"])
```
so it runs even if the permissions are not set properly?

Comment (Cadair) [reply convert.py:28]: It will run anyway right? I have no idea how subprocess handles such things?

**Review by ehsteve — CHANGES_REQUESTED** [review]
I would recommend that this older style be added to the list of styles and not replace the new one.

File: convert.py (hunk 1 above)
Comment (dpshelio) [reply convert.py:28]: No, it doesn't. I've just run a test and I've got:
```python
PermissionError: [Errno 13] Permission denied
```

**Comment by dpshelio** [comment]
I don't understand @ehsteve comment about the sytle...

**Comment by ehsteve** [comment]
@dpshelio I'm suggesting we just keep both styles with and without the linebreak.

**Pull request closed without merging**